Task1/Program.cs

```diff
@@ -10,7 +10,115 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello world!");
+            Task1();
+            Task2();
+            Task3();
+            Task4();
+            Task5(3, 1);
+            Task6(15);
+            Task7();
+            Task8();
+
+        }
+
+        static public void Task1()
+        {
+            int x1 = 6;
+            Console.WriteLine(x1);
+        }
+
+        static public void Task2()
+        {
+            string str1;
```

Comment:
No need to declare the variable before initialization. 
Please place declaration and initialization on the same line.